cmd/analyse.go

```diff
@@ -60,5 +60,9 @@ func runAnalyse(cmd *cobra.Command, args []string) {
 		Host:            viper.GetString("host"),
 		RawManifestFile: manifestFile,
 	}
-	sa.StackAnalyses(requestParams)
+	analysesResult := sa.StackAnalyses(requestParams)
+	if sa.ProcessSummary(analysesResult) {
+		// If Stack has vulnerability, exit with 2 code
```

Comment:
Is this behaviour documented through `--help` switch?

Reply:
Do you mean to include this text in `--help` summary text of `analyse cmd` .?

Reply:
Help text of `analyse cmd` updated